client/src/pages/Login/LoginForm/LoginForm.tsx

```diff
@@ -64,32 +68,42 @@ export default function Login({ handleSubmit }: Props): JSX.Element {
             error={touched.email && Boolean(errors.email)}
             value={values.email}
             onChange={handleChange}
+            variant="outlined"
           />
+          <InputLabel htmlFor="password">
+            <Typography className={classes.label}>PASSWORD</Typography>
+          </InputLabel>
           <TextField
             id="password"
-            label={<Typography className={classes.label}>Password</Typography>}
+            placeholder="Password"
             fullWidth
             margin="normal"
             InputLabelProps={{
               shrink: true,
             }}
             InputProps={{
               classes: { input: classes.inputs },
-              endAdornment: <Typography className={classes.forgot}>Forgot?</Typography>,
+              // endAdornment: <Typography className={classes.forgot}>Forgot?</Typography>,
```

Comment:
Let's remove unused/commented code here (or uncomment if it was a temporary change).